### DISCUSSION TITLE
database connections

### DISCUSSION THREAD

**Discussion by DaStormer** [sequelize/sequelize#13966] [Help & Questions] [answered]
hey, do any of these functions create a new connection _every_ time they're used?

findOne()
create()
update()

I have a postgres database which is limited to 100 concurrent connections. When I use the above functions at scale, I start getting "too many clients already" errors. This leads me to believe that a new connection is created each time. I'm not sure if that's normal

**Comment by papb** [comment, marked as answer]
Can you try [configuring your connection pool](https://sequelize.org/v7/manual/connection-pool.html) to different values and see if it helps? Thanks :)

**Reply by DaStormer** [reply]
This helped, thank you for your prompt solution!

**Reply by DaStormer** [reply]
BTW, would lowering the max affect performance?

**Reply by papb** [reply]
You're welcome, to be honest you got very lucky, I am randomly choosing a few things to look at 😅 

> BTW, would lowering the max affect performance?

I don't know, sorry... I don't understand well this pool thing, just had a hunch that it might help you 😃 I suggest you make some experiments and measure the performance.
